src/main/java/meditracker/ui/Ui.java

```diff
@@ -113,12 +114,22 @@ public static void showInfoMessage(String message) {
     }
 
     /**
-     * Reads user input command.
-     * @return The user input command as a String.
+     * Reads a command from the user input, prompting with "meditracker> ".
+     * This method continuously waits and reads user input until a command is entered.
+     * If there is no more input (end of stream), the method will display an exit message and terminate the program.
+     *
+     * @return The input entered by the user. If the program terminates due to no more input, null is returned.
+     * @throws NoSuchElementException if the input stream is closed and no more tokens are available for retrieval.
```

Comment:
Exception mentioned here is not thrown, but caught and handled. Do remove it from the javadoc.

Reply:
Resolved in 83b7cb039b8f79315a593e8567d8babf556d3865. Thanks!